packages/core/src/components/Avatar/Avatar.tsx

```diff
@@ -154,48 +155,28 @@ export const HvAvatar = ({
   const badgeColor = getColor(badge || "", theme.colors.positive);
 
   return (
-    <StyledContainer
-      className={clsx(avatarClasses.container, classes?.container)}
-      {...others}
-    >
-      <StyledStatus
-        className={clsx(
-          avatarClasses.status,
-          classes?.status,
-          avatarClasses[variant],
-          classes?.[variant],
-          avatarClasses[size],
-          classes?.[size]
-        )}
+    <div className={classes.container} {...others}>
+      <div
+        className={cx(classes.status, classes[variant], classes[size])}
         style={statusInlineStyle}
-        $variant={variant}
-        $size={size}
       >
         {badge && (
-          <StyledBadge
-            className={clsx(avatarClasses.badge, classes?.badge)}
+          <div
+            className={classes.badge}
             style={{ backgroundColor: badgeColor }}
           />
         )}
-        <StyledAvatar
+        <MuiAvatar
           component={component}
-          className={clsx(
-            className,
-            avatarClasses.root,
-            classes?.root,
-            avatarClasses.avatar,
-            classes?.avatar,
-            avatarClasses[size],
-            classes?.[size]
-          )}
+          className={cx(classes.root, classes.avatar, classes[size], className)}
```

Comment:
this is definitely not the `root`... 🤷

Reply:
Yup, it doesn't make much sense to have the `root` and `className` here in my opinion 😅 I didn't change this because this can lead to breaking changes for some users.